simpleflow/job/k8s.py

```diff
@@ -0,0 +1,73 @@
+from base64 import b64encode
+from jinja2 import Template
+import json
+import os
+import yaml
```

Comment:
Maybe add `PyYAML` to the DEPS?

Reply:
Yes. It's implicitly there via the kubernetes dependency but better be explicit.